### PR TITLE
Attempt to fix spurious failures for short_expireafter test on Solaris 9.

### PR DESCRIPTION
**Pull request by jimis**

### CODE REVIEW

**Comment by jimis** [comment]
test-building on Jenkins

**Comment by jimis** [comment]
Our agent seems to have slowed down quite a bit, or the Solaris box needs a reboot @kacf. Disabled the test there, but collected a bunch of logs that I intend to put in a ticket.

File: configure.ac [hunk 1]
```diff
@@ -516,16 +521,21 @@ if test "x$with_libcurl" != xno
...
     AC_CHECK_LIB(curl, curl_global_init,
-      [libcurl_lib_found=yes],
-      [if test "x$with_libcurl" != xcheck; then AC_MSG_ERROR(Cannot find libcurl library); fi])
+      [],
+      [  libcurl_lib_found=no;
+         if test "x$with_libcurl" != xcheck;
+         then
+             AC_MSG_ERROR(Cannot find libcurl library);
+         fi
+      ])
```
Comment (jimis) [inline configure.ac]: @tzz sorry for the delay, this commit should fix the libcurl issue.

**Comment by tzz** [comment]
@jimis I can confirm this branch fixed libcurl, thank you!

**Comment by jimis** [comment]
Thank you @tzz for detecting and reporting it!

**Pull request merged**